### PR TITLE
Fix Markdown on documentation for install-kubectl

### PR DESCRIPTION
**Pull request by dduportal**
Hello dear maintainer and reviewers.

Thanks for this awesome documentation!

While browsing it, I found a markdown rendering issue in the section "Install with Powershell from PSGallery" in the page [Install and Set Up kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).

I took the opportunity to apply a few code styling to the script/command names of the "Update" bullet, as well as adapting the the variable to Windows's style (given the context of this section).

I try to follow the https://kubernetes.io/docs/contribute/style/style-guide/ , but don't hesitate to point me any error or misleading I could have done in this proposition.

Only got  an issue with the CLA: it appears to be in 404, as well as the signing form (from the https://kubernetes.io/docs/contribute/start/#sign-the-cla , both links are in 404 when writing these lines: https://kubernetes.io/docs/imported/community/guide/ and https://kubernetes.io/docs/imported/community/guide/#sign-the-cla.

Thanks in advance for your time!

### CODE REVIEW

**Comment by bradtopol** [comment]
Hi @dduportal Thank you for your contribution. Please sign the CLA so we can accept your PR for review.

**Comment by dduportal** [comment]
Hello @bradtopol , it has been done: CLA is signed.

Looks like the bots saw the change: the check is passing for cla and label had been added to the PR.

**Comment by kbarnard10** [comment]
@dduportal Can you resolve the merge conflicts?

**Comment by dduportal** [comment]
Hello @kbarnard10 ! The conflict had been resolved. Any feedback welcome, because I am not an English native so my writing might not be very good.

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -100,24 +100,22 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
 If you are on Windows and using [Powershell Gallery](https://www.powershellgallery.com/) package manager, you can install and update kubectl with Powershell.
 
 To install:
-* Run the installation commands (making sure to specify a DownloadLocation):
+* Run the installation commands (making sure to specify a `DownloadLocation`):
```
Comment (bradtopol) [inline content/en/docs/tasks/tools/install-kubectl.md]: Please update the * to instead follow the structure shown in the ## Install with Chocolatey on Windows section below (using numbers instead).  The * is not rendering properly

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 2]
```diff
@@ -100,24 +100,22 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
...
     
     {{< note >}}
-    **Note:** If you do not specify a DownloadLocation, kubectl will be installed in the user's temp Directory.
+    **Note:** If you do not specify a `DownloadLocation`, `kubectl` will be installed in the user's temp Directory.
     {{< /note >}}
-    The installer creates $HOME/.kube and instructs it to create a config file
+    The installer creates `$HOME/.kube` and instructs it to create a config file
+
 To update:
 * Run the update commands:
```
Comment (bradtopol) [inline content/en/docs/tasks/tools/install-kubectl.md]: Please update this one as well:  Please update the * to instead follow the structure shown in the ## Install with Chocolatey on Windows section below (using numbers instead).  The * is not rendering properly

**Comment by bradtopol** [comment]
Hi @dduportal the * you added are not rendering properly. Please see my inline comments on how to fix this and have the section look like similar sections in the document.  Thanks!

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 3]
```diff
@@ -99,26 +99,29 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
...
+2. Test to ensure the version you installed is sufficiently up-to-date:
 
     ```
-    re-run Install-Script to update the installer
-    re-run install-kubectl.ps1 to install latest binaries
+    kubectl version
     ```
 
+{{< note >}}
+**Note:** Updating the installation consists in running again the two commands of the step 1.
```
Comment (bradtopol) [inline content/en/docs/tasks/tools/install-kubectl.md]: This sentence is not grammatically correct.  Perhaps:
Updating the installation is performed by rerunning the two commands listed in step 1.

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 4]
```diff
@@ -99,26 +99,29 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
...
-    The installer creates $HOME/.kube and instructs it to create a config file
-To update:
-* Run the update commands:
+{{< note >}}
+**Note:** If you do not specify a `DownloadLocation`, `kubectl` will be installed in the user's temp Directory.
+{{< /note >}}
+    
+The installer creates `$HOME/.kube` and instructs it to create a config file
+
+2. Test to ensure the version you installed is sufficiently up-to-date:
```
Comment (bradtopol) [inline content/en/docs/tasks/tools/install-kubectl.md]: This is listed as a 2 but is rendering as a 1.  It needs to be fixed to remove vagueness.

**Comment by bradtopol** [comment]
Hi @dduportal there are still some issues that need to be fixed. See my inline comments. Thanks for working on this!

**Comment by dduportal** [comment]
Hello @bradtopol ! 

Your feedback had been taken in account. I had to experiment a bit to figure out how to achieve the correct rendering, and squashed the "experiments" commits.

The bullet rendering is behaving as expected, as well as the notes: the `{{note}}` markups had to be indented to be part of the bullet block (and not reseting the bullet numbering).
However, the content of these notes should NOT be indented, to avoid rendering it all bold.

**Comment by tengqm** [comment]
/lgtm
Preview looks good now. Defer to Brad for approval.

**Comment by bradtopol** [comment]
Great job!
/lgtm
/approve

**Pull request merged**